addons/sourcemod/translations/goomba.phrases.txt

```diff
@@ -6,22 +6,22 @@
 		"fr"			"{olive}>> {teamcolor}{1} {default}a stompé {olive}{2} {default}!"
 		"en"			"{olive}>> {teamcolor}{1} {default}goomba stomped {olive}{2} {default}!"
 		"ru"			"{olive}>> {teamcolor}{1} {default}приземлился на {olive}{2} {default}!"
-		"pt"			"{olive}>> {teamcolor}{1} {default}deu goomba stomp em {olive}{2} {default}!"
+		"pt"			"{olive}>> {teamcolor}{1} {default}deu um goomba stomp em {olive}{2} {default}!"
 	}
 	"Goomba Stomp TeamKill"
 	{
 		"#format"		"{1:N},{2:N}"
 		"fr"			"{olive}>> {teamcolor}{1} {default}a stompé {teamcolor}{2} {default}!"
 		"en"			"{olive}>> {teamcolor}{1} {default}goomba stomped {teamcolor}{2} {default}!"
 		"ru"			"{olive}>> {teamcolor}{1} {default}приземлился на {teamcolor}{2} {default}!"
-		"pt"			"{olive}>> {teamcolor}{1} {default}deu goomba stomp em {teamcolor}{2} {default}!"
+		"pt"			"{olive}>> {teamcolor}{1} {default}deu um goomba stomp em {teamcolor}{2} {default}!"
 	}
 	"Victim Stomped"
 	{
-		"fr"			"Vous avez été stompé !"
```

Comment:
In french, there's a space between the last word and question/exclamation mark.

Reply:
Fixed.